src/EFCore.PG/Metadata/INpgsqlIndexAnnotations.cs

```diff
@@ -20,5 +20,7 @@ public interface INpgsqlIndexAnnotations : IRelationalIndexAnnotations
         /// https://www.postgresql.org/docs/current/static/indexes-opclass.html
         /// </remarks>
         IReadOnlyList<string> Operators { get; }
+
+        IReadOnlyList<string> IncludeProperties { get; }
```

Comment:
I'm not convinced that `IncludeProperties` is the best name for this. It is, after all, column names and not property names (in case they're not the same). I went with this name purely for consistency with the SQL Server naming. Not sure if that's a goal or not...

Reply:
What do you think about `NonKeyColumns` (or `NonKeyProperties`) with an XML doc like _a list of columns which will be included in the index as non-key columns_?

Reply:
Maybe `IncludedColumns` since they are included into the index, or `IncludedProperties`? So it won't sound like a method name.

Reply:
I'm concerned that `IncludeColumns` or `IncludedColumns` could be mistaken to mean "the indexed columns" rather than "the non-key columns".

Reply:
The [documentation](https://www.postgresql.org/docs/current/sql-createindex.html) mostly uses "non-key columns" to name them. Therefore, I would prefer to have the same name.

Reply:
Note that `IncludeProperties` is what the EF Core team uses upstream, so I'd vote for keeping that, it's important to keep a consistent terminology across the providers etc.

Reply:
Cross-provider consistency is a good reason to keep it. As long as we have an XML doc that mentions non-key columns, no objections here.

Reply:
So we're sticking with the current naming?

Reply:
Yes